docs/website/docs/dlt-ecosystem/verified-sources/notion.md

```diff
@@ -0,0 +1,124 @@
+    ```bash
+    dlt init notion bigquery
+    ```
+    
+    This command will initialize your verified source with Notion and creates a pipeline with BigQuery as the destination. If you'd like to use a different destination, simply replace `bigquery` with the name of your preferred destination. You can find supported destinations and their configuration options in our [documentation](https://dlthub.com/docs/destinations/duckdb)
+    
+3. After running this command, a new directory will be created with the necessary files and configuration settings to get started.
+    
+    ```toml
+    notion_source
+    ├── .dlt
+    │   ├── config.toml
+    │   └── secrets.toml
+    ├── notion
+    │   └── helpers
+    │   └── __init__.py
+    │   └── helpers.py
+    │   └── README.md
+    │   └── settings.py
+    ├── .gitignore
+    ├── requirements.txt
+    └── notion_pipeline.py
+    ```
+    
+
+## **Add credential**
+
+1. Inside the **`.dlt`** folder, you'll find a file called “_secrets.toml_”, which is where you can securely store your access tokens and other sensitive information. It's important to handle this file with care and keep it safe.
+    
+    Here's what the file looks like:
+    
+    ```toml
+    # Put your secret values and credentials here
+    # Note: Do not share this file and do not push it to GitHub!
+    [source.notion]
+    api_key = "set me up!" # Notion API token (e.g. secret_XXX...)
+    
+    [destination.bigquery.credentials] # The credentials required will change based on the destination
+    project_id = "set me up!" # GCP project ID
+    private_key = "set me up!" # Unique private key (including `BEGIN and END PRIVATE KEY`)
+    client_email = "set me up!" # Service account email
+    location = "set me up!" # Project location (e.g. “US”)
+    
+    ```
+    
+2. Replace the value of `api_key` with the one that [you copied above](notion.md#grab-api-credentials). This will ensure that your data-verified source can access your notion resources securely.
+3. Next, follow the instructions in [Destinations](https://dlthub.com/docs/destinations/duckdb) to add credentials for your chosen destination. This will ensure that your data is properly routed to its final destination.
+
+## Run the pipeline example
+
+1. Install the necessary dependencies by running the following command:
+    
+    ```bash
+    pip install -r requirements.txt
+    ```
+    
+2. Now the pipeline can be run by using the command:
+    
+    ```bash
+    python3 notion_pipeline.py
+    ```
+    
+3. To make sure that everything is loaded as expected, use the command:
+    
+    ```bash
+    dlt pipeline <pipeline_name> show
+    ```
+    
+    For example, the pipeline_name for the above pipeline example is `notion`, you may also use any custom name instead.
+    
+
+## Customizations
+
+To load data to the destination using `dlt`, you have the option to write your own methods. It is important to note that this method retrieves data from the databases.
```

Comment:
> It is important to note that this method retrieves data from the databases.

i would change this sentence.  "this method retrieves data from all existed databases in your Notion account"

> It is important to note 

it duplicates line 122. rephrase it please.